### PR TITLE
Initial Refactors

### PR DESCRIPTION
**Pull request by v3ceban**
- Changed Nav dropdown to use shadcn/ui component
- Made Nav a server component, moved client logic into separate components
- Moved css files under @/styles
- Added super_admin to protected routes
- Changed protected routes redirects to "/", resolving infinite redirect to "/success/new-user"
  - This should be changed to "/login" when homepage gets a different purpose
- Added "/about" and "/privacy-policy" that are linked in footer
- Added user profile picture to session token
- Extracted user profile picture url from google into db
- Changed Nav logo to look like the one in Canva
- Changed footer copyright symbol to html-safe one
  - @marleyyvon do we even need it? we're under GPLv3.0, which is a copyleft license... pls review license terms and revise

### CODE REVIEW

**Comment by v3ceban** [comment]
Also added predev npm isntall script to ensure you have all necessary dependencies when you start your dev server

**Review by marleyyvon — DISMISSED** [review]
We can remove that, will update on the other branch shortly.

**Review by miaadele — DISMISSED** [review]

File: src/components/Navbar/Navbar.jsx [hunk 1]
```diff
@@ -1,72 +1,88 @@
...
 import Link from "next/link";
+import {
+  DropdownMenu,
+  DropdownMenuContent,
+  DropdownMenuItem,
+  DropdownMenuLabel,
+  DropdownMenuSeparator,
+  DropdownMenuTrigger,
+} from "@/components/ui/dropdown-menu";
+import { Login, Logout } from "./AuthButtons";
```
Comment (jcisneros1999) [inline src/components/Navbar/Navbar.jsx:13]: Question: Should we make all paths use the @/ convention for consistency?

**Review by jcisneros1999 — CHANGES_REQUESTED** [review]

File: src/components/Navbar/Navbar.jsx (hunk 1 above)
Comment (jcisneros1999) [reply src/components/Navbar/Navbar.jsx:13]: Also feel free to tell me to kick rocks if its not important lol. I got no strong opinion on this, consistency just helps make the project feel like it is cohesive.

Comment (miaadele) [reply src/components/Navbar/Navbar.jsx:13]: Sure, I can change it

**Review by vaniaggarwal — APPROVED** [review]
looks good to me

**Review by marleyyvon — APPROVED** [review]

**Review by miaadele — APPROVED** [review]

**Pull request merged**